learning/prompts/documentation-lookup/17_advanced_ai_ml.md

```diff
@@ -0,0 +1,18 @@
+Prompt:
+What are AI and ML capabilities in Apache Beam?
+Response:
+Apache Beam has a number of built-in [AI and ML capabilities](https://beam.apache.org/documentation/ml/overview/) that enable you to:
```

Comment:
Apache Beam has several built-in [AI and ML capabilities](https://beam.apache.org/documentation/ml/overview/) that enable you to: